src/providers/keys/client.py

```diff
@@ -58,15 +68,27 @@ def _get_with_blockstamp(self, url: str, blockstamp: BlockStamp, params: dict |
     @lru_cache(maxsize=1)
     def get_used_lido_keys(self, blockstamp: BlockStamp) -> list[LidoKey]:
         """Docs: https://keys-api.lido.fi/api/static/index.html#/keys/KeysController_get"""
-        return list(map(lambda x: LidoKey.from_response(**x), self._get_with_blockstamp(self.USED_KEYS, blockstamp)))
+        data = list(map(lambda x: LidoKey.from_response(**x), self._get_with_blockstamp(self.USED_KEYS, blockstamp)))
+        self._check_used_keys(data)
```

Comment:
Can we check an impact on current processing? I mean time. Because we have hundreds of thousands keys on mainnet.